capten/config-worker/internal/crossplane/config_provider_update.go

```diff
@@ -0,0 +1,105 @@
+package crossplane
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	fileutil "github.com/kube-tarian/kad/capten/config-worker/internal/file_util"
+	"github.com/kube-tarian/kad/capten/model"
+	agentmodel "github.com/kube-tarian/kad/capten/model"
+	"github.com/pkg/errors"
+)
+
+func (cp *CrossPlaneApp) configureConfigProviderUpdate(ctx context.Context, req *model.CrossplaneClusterUpdate) (status string, err error) {
+	logger.Infof("configuring config provider %s update", req.ManagedClusterName)
+
+	x, _ := json.Marshal(req)
+
+	fmt.Println("configureConfigProviderUpdate request")
+
+	fmt.Println(string(x))
+
```

Comment:
why all the code is duplicated from other action.

it just need to synch aws-k8s-proivder app